### ISSUE TITLE
Support .NET 6/7

### ISSUE THREAD

**Issue by zanYObyte** [BranchMetrics/xamarin-branch-deep-linking-attribution#152]
### Describe the feature

With the pending sunsetting of Xamarin by Microsoft, and their decision to not suppport iOS 17 anymore, we are looking for ways to migrate to .NET over Xamarin.
However, as far as I have understood, there is no .NET 6/7 version of the Branch SDK available yet.
Is there any indication if and when this will be made available?

**Comment by gerardvanderkruijs** [comment]
Can we get an update on this feature request please?

**Comment by echo-branch** [comment]
The ticket is in the current sprint, however it may slip into the next sprint depending on how progress looks.

**Comment by gerardvanderkruijs** [comment]
Thanks for the update! Really nice to know development is in progress and that we can expect an updated version soon 🙌

**Comment by echo-branch** [comment]
@gerneio You'll likely want the SDK, unless you're at an organization setup for S2S integration. Work is still in progress on getting MAUI support working properly. There was a issue with the latest Xcode 14.3 breaking MAUI, but a fix was just released.

**Comment by gerneio** [comment]
Thanks for confirming @echo-branch 

Honestly, not quite sure what I'm needing, still new to all this. All I know is trying to figure out how to implement deep linking and deferred deep linking for iOS/Android mobile apps in a platform/version agnostic way has been a royal pain. Which has led me to `branch.io` for the hope of a simplistic end-to-end solution. 

Really, I just need a landing page that redirects to the app (on user button click), and does all the "smart" stuff of device/platform detection, and recognizing if app is already installed or not. I have already developed that landing page (in house) and it works quite well, but it's a two button solution and the user needs to know to click the right one (one to open app w/ custom uri scheme and another to open platform specific app store), but I need to "share" a sign up code across the browser-to-app boundary. Working perfectly when using custom uri scheme, but if user needs to first install the app, I'm having a hard time working through the transferring state through the installation process (i.e. deferred deep linking?). Seems like Google has this `InstallReferrerClient` that can be used to get sate about the install process, but still trying to reason through how to even pass along that state in the first place. From my research, iOS doesn't really support such a thing (at least a clean native way?). The goal would be to avoid having the user have to revisit the landing web page, if at all possible. 

From my research thus far, `branch.io` seems top at the list for supporting all these complicated, version/platform dependent features, in hopefully a common abstraction layer. But honestly it's a little difficult to find the parts that I need when digging through the `branch.io` docs (there's just a lot of parts and pieces, considering what appears to be a simple project, for good reason I know), and I'm assuming I will need to wait on the MAUI specific SDK to even integrate with `branch.io` in the first place (at least for the MAUI app side of things).

Looking forward to testing out the MAUI SDK once yall have it available! ❤️

**Comment by zanYObyte** [comment]
@echo-branch Just out of curiosity, how long are your sprints?:D

**Comment by echo-branch** [comment]
Branch Xamarin SDK 9.0 has finally graduated from pre-release.

Some of the integration steps have changed and docs updates are in progress.

https://github.com/BranchMetrics/xamarin-branch-deep-linking-attribution/releases/tag/9.0.0
